documentation/release_notes.markdown

```diff
@@ -4,87 +4,271 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.0
```

Comment:
I'm OK with that, but then I'd like to rename the file between branches because it's just going to be a ton of conflicts.  Sort of like the ps-version in project.clj, but way worse.